test/io.jl

```diff
@@ -148,12 +148,26 @@ function test_json_io()
     return nothing
 end
 
+function test_export_ptdf()
+    data = make_basic_network(pglib("14_ieee"))
+    solver = OPT_SOLVERS[OPFGenerator.EconomicDispatch]
+    opf = OPFGenerator.build_opf(OPFGenerator.EconomicDispatch, data, solver)
```

Comment:
Maybe we can compute the PTDF matrix directly here, instead of having to build the ED model?

Reply:
Will add a function `export_ptdf(data::Dict{String,Any}, filepath)` that takes a PowerModels dict instead of an OPFModel